### PR TITLE
Assistant Registration [Ready to Merge]

### PR DESCRIPTION
**Pull request by AnturK**
Allows other jobs to register assistants to them, giving them related basic access on the spot. The intent is to make some use of shambling hordes of greyshirts. Accesses granted are up to change. Will have to add some notification for Sec Records/Crew Manifest to make tracking accesses easier. Pretty radical change so feel free to bash the idea.

### CODE REVIEW

**Comment by Petethegoat** [comment]
This is a really cool idea!

I'm gonna make a PR to your branch to tidy up a few bits, but this looks fine overall. It would be cool if you could swipe their cards with your card inside a PDA, if that makes sense.

File: code/game/jobs/job/job.dm [hunk 1]
```diff
@@ -7,6 +7,9 @@
 	var/list/minimal_access = list()		//Useful for servers which prefer to only have access given to the places a job absolutely needs (Larger server population)
 	var/list/access = list()				//Useful for servers which either have fewer players, so each person needs to fill more than one role, or servers which like to give more access, so players can't hide forever in their super secure departments (I'm looking at you, chemistry!)
 
+	//Assistant acess : acess granted to job's assistants
```
Comment (Iamgoofball) [inline code/game/jobs/job/job.dm:10]: change acess to access

**Comment by Iamgoofball** [comment]
I love you

:+1: x :100:

**Comment by Petethegoat** [comment]
PR created: https://github.com/AnturK/NTstation13/pull/1

**Comment by AnturK** [comment]
Merged the tidy up, will make the feedback thread maybe something interesting will pop up that could be included in this. Will check the id in pda later today.

**Comment by AnturK** [comment]
Now you can remove the registration at HoP's console and PDA swiping works.

**Comment by AnturK** [comment]
Well forums reaction is pretty positive so i guess it's more or less ready to merge.

**Pull request merged**